### ISSUE TITLE
Context menu for fixing current file

### ISSUE THREAD

**Issue by fluxsauce** [AtomLinter/linter-eslint#955]
## Issue Type
Feature Request

## Issue Description

This was previously discussed in https://github.com/AtomLinter/linter-eslint/issues/940 but as a support request. The explanation (use the command palette), while accurate isn't user friendly. Fixing files on-demand is a common use case, and something supported by other editors like WebStorm.

In atom, the hacky way to do it is to edit the Init Script

```cs
atom.contextMenu.add {
  'atom-text-editor, .overlayer': [
    { type: 'separator' },
    {
      label: 'ESLint Fix',
      command: 'linter-eslint:fix-file'
    }
  ]
}
```

![screen shot 2017-06-30 at 1 51 51 pm](https://user-images.githubusercontent.com/976391/27753647-4cbd50d2-5d9b-11e7-8120-7a80cccfce99.png)

I don't know enough about `shouldDisplay` and Atom to make it only show for JavaScript files, but that's the API to use.

**Comment by Arcanemagus** [comment]
Well, in trying to prove it was impossible to do this right, I found a way to do it 😛.

Any objection to adding this @IanVS?

**Comment by Arcanemagus** [comment]
Released in v8.3.0. 🎉

**Comment by fluxsauce** [comment]
Fantastic, thank you!
